haystack/nodes/connector/crawler.py

```diff
@@ -231,8 +231,17 @@ def run(  # type: ignore
         overwrite_existing_files: Optional[bool] = None,
         return_documents: Optional[bool] = False,
         id_hash_keys: Optional[List[str]] = None,
+<<<<<<< HEAD
```

Comment:
This seems to be an unresolved merge conflict that needs to be resolved before merging.

Reply:
Apparently you were faster in fixing this than my review :D